app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt

```diff
@@ -1,22 +1,100 @@
 package com.intive.picover.parties.view
 
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Card
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.res.stringResource
-import com.intive.picover.R
+import androidx.compose.ui.platform.LocalLifecycleOwner
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavHostController
+import com.intive.picover.common.loader.PicoverLoader
+import com.intive.picover.main.theme.Typography
+import com.intive.picover.parties.model.Party
+import com.intive.picover.parties.state.PartiesState
+import com.intive.picover.parties.viewmodel.PartiesViewModel
 
 @Composable
-fun PartiesScreen() {
-	Box(
-		modifier = Modifier.fillMaxSize(),
+fun PartiesScreen(
+	viewModel: PartiesViewModel,
+	navController: NavHostController,
+) {
+	LocalLifecycleOwner.current.lifecycle.addObserver(viewModel)
+	when (val state = viewModel.parties.value) {
+		is PartiesState.Loading -> {
+			PicoverLoader()
+		}
+
+		is PartiesState.Loaded -> {
+			LoadedContent(
+				parties = state.parties,
+				openDetails = {
+					navController.navigate("partyDetails/$it")
```

Comment:
I think that we should pass a lambda here instead of passing `navController` and navigating, but navigation is still a thing that we need to discuss so we can leave it for now.